### PR TITLE
Refactor food collision

### PR DESCRIPTION
**Pull request by karinamzalez**
- refactor food collision functions 
- make them more versatile 
  - rename functions and variables appropriately 
- all 57 tests still passing

### CODE REVIEW

File: test/health-food-test.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const chai = require('chai');
 const assert = chai.assert;
-const stub = require('./support/stub');
+// const stub = require('./support/stub');
```
Comment (deborahleehamel) [inline test/health-food-test.js:3]: comment to remove

File: lib/random-junk-food.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-    {x: x, "type": "pancakes", "color": "saddlebrown", "score": "5"},
-    {x: x, "type": "white_bread", "color": "gray", "score": "5"}
+    {x: x, "type": "burger", "color": "saddlebrown", "score": "-5"},
+    {x: x, "type": "hotdog", "color": "tan", "score": "-5"},
+    {x: x, "type": "cake", "color": "black", "score": "-5"},
+    {x: x, "type": "friedChicken", "color": "dimgray", "score": "-5"},
+    {x: x, "type": "pizza", "color": "salmon", "score": "-5"},
+    {x: x, "type": "corndog", "color": "tan", "score": "-5"},
+    {x: x, "type": "pancakes", "color": "saddlebrown", "score": "-5"},
+    {x: x, "type": "white_bread", "color": "gray", "score": "-5"}
```
Comment (chompasina) [inline lib/random-junk-food.js:10]: Good thinking to change this to negative scores for the total score function.

**Comment by chompasina** [comment]
Approved!

File: lib/junk-food.js [hunk 3]
```diff
@@ -1,14 +1,21 @@
...
+  this.width    = settings.width || 40;
+  this.color    = settings.color;
+  this.type     = settings.type;
+  this.speed    = settings.speed || 0.5;
+  this.ctx      = settings.ctx;
   this.overlord = settings.overlord;
   this.score    = settings.score;
 }
 
+JunkFood.prototype.move = function() {
```
Comment (deborahleehamel) [inline lib/junk-food.js:14]: I like this as a prototype method for move.

**Pull request merged**